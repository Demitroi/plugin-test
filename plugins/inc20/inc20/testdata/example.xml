<articulo nombre="n64">
    <precio1>10</precio1>
    <precio2>15.25</precio2>
    <precio3>20</precio3>
</articulo>