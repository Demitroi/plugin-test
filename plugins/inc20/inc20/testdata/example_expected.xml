<articulo nombre="n64">
    <precio1>12</precio1>
    <precio2>18.3</precio2>
    <precio3>24</precio3>
</articulo>